src/controllers/files/move_a_file/_handler.py

```diff
@@ -0,0 +1,30 @@
+from src.config.soap_client import soap_client
+from flask import request
+
+
+def file_move_handler(token, file_uuid):
+    try:
+        data = request.get_json()
```

Comment:
You should use `json.loads()` instead of `get_json()`. That's because `json.loads()` throws a `ValueError` exception when no JSON is received.